depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml

```diff
@@ -7,7 +7,7 @@ files:
     size: 138536
     sha256: bd03684c902e99df3156d331167d94d965ad12193f2a8e26903be27ac17414fd
     source: https://blobconverter.s3.us-west-2.amazonaws.com/deeplabv3.xml
-  - name: FP16/deeplabv3p_person.bin
+  - name: FP16/deeplabv3p_person.xml
```

Comment:
Is this correct?

Reply:
Also we might want to just move the model to depthai-model-zoo and use blobconverter to download/convert it

Reply:
It's already in depthai zoo, I've replaced it with depthai's model.